src/libm/sleefsimdsp.c

```diff
@@ -256,17 +258,113 @@ static INLINE CONST vfloat vldexp3_vf_vf_vi2(vfloat d, vint2 q) {
 
 EXPORT CONST vfloat xldexpf(vfloat x, vint2 q) { return vldexp_vf_vf_vi2(x, q); }
 
+#ifdef ENABLE_SVE
+typedef __sizeless_struct {
+  vfloat d;
+  vint2 i;
```

Comment:
Shouldn't this be `vint i;` and not `vint2 i;`?

Reply:
vint2 is correct here

Reply:
my bad, sorry